twiliobot/main.py

```diff
@@ -0,0 +1,72 @@
+import os
+import logging
+import json
+import asyncio
+from datetime import datetime
+
+from quart import Quart, request, jsonify
+from Akvo_rabbitmq_client import rabbitmq_client
+from twiliobot_client import twiliobot_client
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+app = Quart(__name__)
+
+
+RABBITMQ_QUEUE_TWILIOBOT_REPLIES = os.getenv('RABBITMQ_QUEUE_TWILIOBOT_REPLIES')
+RABBITMQ_QUEUE_USER_CHATS = os.getenv('RABBITMQ_QUEUE_USER_CHATS')
+
+
+# Helper function to format Twilio messages for RabbitMQ
+def twilio_POST_to_queue_message(values: dict) -> str:
+    iso_timestamp = datetime.now().isoformat()
+    queue_message = {
+        'id': values['MessageSid'],
+        'timestamp': iso_timestamp,
+        'platform': 'WHATSAPP',
+        'from': {
+            'phone': values['From'].split(':')[1],
```

Comment:
Hi @kjkoster , 
The TwilioBot will receive the phone number from the backend, right? Since we have validation in the backend, should I use the same Pydantic validation here?